### PR TITLE
chore: upgrade java 8 and 11 images

### PR DESCRIPTION
**Pull request by clemstoquart**

### CODE REVIEW

**Comment by Crow-EH** [comment]
Hi @clemstoquart,

Could you upgrade jdk 8 in the same PR please ?
8u222 is now available as debian-slim :D !

**Comment by clemstoquart** [comment]
@Crow-EH Sure. Done !

**Comment by Crow-EH** [comment]
@clemstoquart Can you resolve the conflicts please ?

**Comment by clemstoquart** [comment]
@Crow-EH Done

File: CHANGELOG.md [hunk 1]
```diff
@@ -34,6 +34,7 @@ Versions
 * Use only major Node version to name node images
 * Add Node 12
 * Upgrade ScoutSuite to 5.3.3
+* Upgrade Java: 8u222, 11.0.4
```
Comment (mremi) [inline CHANGELOG.md]: Can you please amend the existing line 11 instead?

Comment (clemstoquart) [reply CHANGELOG.md]: Done :)

**Pull request merged**